src/SignalR.Orleans/Core/GrainExtensions.cs

```diff
@@ -13,7 +13,7 @@ public static class GrainSignalRExtensions
     {
         public static async Task SendSignalRMessage(this IConnectionGrain grain, string methodName, params object[] message)
         {
-            var invocationMessage = new InvocationMessage(Guid.NewGuid().ToString(), nonBlocking: true, target: methodName, arguments: message);
+            var invocationMessage = new InvocationMessage(target: methodName, argumentBindingException: null, arguments: message);
```

Comment:
why did you remove the invocationId?

Reply:
Looking through SignalR's PRs I found talk of removing invocation ID... https://github.com/aspnet/SignalR/pull/1218/files

specifically 

https://github.com/aspnet/SignalR/commit/00a6dc983ac1df1ddc51c968c060bff8f45e5b2e#diff-78e1f2ad8780817477e28d3ec3b11ff6

I might have a misunderstanding though.... It appears that they've change it yet again pending next release.
https://github.com/aspnet/SignalR/pull/1843/files

Reply:
Thanks for your answers, in one of the PR's you've linked there is below (sharing it for general knowledge):
Non-Blocking Invocations
+Invocations can be sent without an `Invocation ID` value. This indicates that the invocation is "non-blocking", and thus the caller does not expect a response. When a Callee receives an invocation without an `Invocation ID` value, it **must not** send any response to that invocation.